src/JanrainCaptureApi.php

```diff
@@ -122,7 +124,7 @@ public function __construct(
   /**
    * {@inheritdoc}
    */
-  public function authenticate(string $auth_code, string $redirect_uri): UserInterface {
+  public function authenticate(string $auth_code, string $redirect_uri): ?Url {
```

Comment:
Why are you changing the return type?

Reply:
That's true return value is ignored in controller, it shouldn't cause any issues.

See https://github.com/EdisonLabs/janrain_capture/blob/8.x-1.x/src/Controller/AuthenticationController.php#L110